test/screens/v2/widget_instance/departures_test.exs

```diff
@@ -260,6 +284,80 @@ defmodule Screens.V2.WidgetInstance.DeparturesTest do
       assert %{type: :no_data_section, text: expected_text} ==
                Departures.serialize_section(section, dup_screen, now, true)
     end
+
+    test "serializes sections with destination grouping", %{
+      bus_shelter_screen: bus_shelter_screen,
+      now: now
+    } do
+      rows = [
+        %Departure{
+          prediction: %Prediction{
+            departure_time: ~U[2020-01-01T00:01:10Z],
+            route: %Route{id: "Green-E", type: :subway},
+            trip: %Trip{headsign: "Medford/Tufts", direction_id: 1},
+            stop: %Stop{}
+          }
+        },
+        %Departure{
+          prediction: %Prediction{
+            departure_time: ~U[2020-01-01T00:01:10Z],
+            route: %Route{id: "Green-D"},
+            trip: %Trip{headsign: "Government Center", direction_id: 1},
+            stop: %Stop{}
+          }
+        },
+        %Departure{
+          prediction: %Prediction{
+            departure_time: ~U[2020-01-01T00:01:10Z],
+            route: %Route{id: "Green-C", type: :subway},
+            trip: %Trip{headsign: "Cleveland Circle", direction_id: 0},
+            stop: %Stop{}
+          }
+        },
+        %Departure{
+          prediction: %Prediction{
+            departure_time: ~U[2020-01-01T00:01:10Z],
+            route: %Route{id: "Green-C", type: :subway},
+            trip: %Trip{headsign: "Cleveland Circle", direction_id: 0},
+            stop: %Stop{}
+          }
+        },
+        %Departure{
+          prediction: %Prediction{
+            departure_time: ~U[2020-01-01T00:01:10Z],
+            route: %Route{id: "Green-D", type: :subway},
+            trip: %Trip{headsign: "Riverside", direction_id: 0},
+            stop: %Stop{}
+          }
+        }
+      ]
+
+      section = %NormalSection{
+        rows: rows,
+        layout: %Layout{},
+        header: %Header{title: "Section Header"},
+        grouping_type: :destination
+      }
+
+      assert %{
+               type: :normal_section,
+               header: %{title: "Section Header", arrow: nil, read_as: nil},
```

Comment:
The header isn't really relevant to this test case; I think we could remove this from the pattern match.